AutomatedTesting/Gem/PythonTests/prefab/Prefab_BasicWorkflow_CreateAndDeletePrefab.py

```diff
@@ -0,0 +1,36 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+def Test():
+
+    CAR_PREFAB_FILE_NAME = 'CAR_prefab'
+
+    def run_test():
```

Comment:
Why are you nesting another function? This is not required

Reply:
Will fix it.

Reply:
Still need to use nesting in the prefab reparenting test since the test is using async.

Reply:
Fixed.